cmd/dep/init.go

```diff
@@ -238,14 +181,19 @@ func (cmd *initCommand) Run(ctx *dep.Ctx, args []string) error {
 	return nil
 }
 
-// contains checks if a array of strings contains a value
-func contains(a []string, b string) bool {
-	for _, v := range a {
-		if b == v {
-			return true
-		}
+func getDirectDependencies(root, cpr string) (*pkgtree.PackageTree, map[string]bool, error) {
```

Comment:
Sure thing, I'll switch it to a value.

What's your preferred style to follow when a function returns a result and an error, such as `Foo() (FooResult, error)`? I am not sure when you would prefer me to use a pointer vs. a value. My usual way is to use nillable types, returning nils when an error occurred.